### ISSUE TITLE
Could not run 'llm-server' with ollama (openchat/llama2)

### ISSUE THREAD

**Issue by jvgrootveld** [openchatai/OpenCopilot#497]
**Describe the bug**
Could not run 'llm-server' locally with Ollama. The docker container sits in a restart loop.

Container log shows error:
```
ImportError: Could not import llama-cpp-python library. Please install the llama-cpp-python library to use this embedding model: pip install llama-cpp-python
```

**To Reproduce**
1. Clone repository 'git clone https://github.com/openchatai/opencopilot.git'
2. Set environment variables:
```
OPENAI_API_TYPE=llama2
EMBEDDING_PROVIDER=openchat
```
3. Install [Ollama](https://ollama.ai)
4. Pull LLM models: `ollama pull openchat && ollama pull llama2`
5. Run 'make install' (or 'make install-arm')

**Expected behavior**

**Screenshots**
On 'make install-arm':
![image](https://github.com/openchatai/OpenCopilot/assets/1201849/6c8044be-15d3-44fd-869d-abcdac3e336f)

Running containers:
![image](https://github.com/openchatai/OpenCopilot/assets/1201849/0edcf3a2-3971-451e-8922-54239f67d1a1)

Log of 'llm-server':
![image](https://github.com/openchatai/OpenCopilot/assets/1201849/5140549f-e4ba-4814-bdee-f2337e41bfd1)

**Desktop (please complete the following information):**
 - OS: MacOS Sonoma 14
 - Browser: -
 - Version: -

**Additional context**

**Comment by jvgrootveld** [comment]
Found out that the 'OPENAI_API_TYPE' value 'llama2' does not work. I also noticed that the `llm = get_llm()` is used 3 times but not used in the code. A llm is used via `chat = get_chat_model()`.

TLDR. To get OpenCopilot working with locally run LLM via Ollama set these variables in the `.ENV` file:

```
OPENAI_API_KEY=unused
CHAT_MODEL=openchat
EMBEDDING_PROVIDER=openchat
LOCAL_IP=http://host.docker.internal
VECTOR_SIZE=4096
```

**Comment by codebanesr** [comment]
Hey @jvgrootveld ,
thanks for mentioning it. Were you able to get it to work ? If not, I can help you with it

**Comment by jvgrootveld** [comment]
@codebanesr Yes I got it working with the variables in the `.env` file I mentioned above. I looked at the code myself to figure this out, good that it's now part of the readme!

**Comment by hugo921** [comment]
> Found out that the 'OPENAI_API_TYPE' value 'llama2' does not work. I also noticed that the `llm = get_llm()` is used 3 times but not used in the code. A llm is used via `chat = get_chat_model()`.
> 
> TLDR. To get OpenCopilot working with locally run LLM via Ollama set these variables in the `.env` file:
> 
> ```
> OPENAI_API_KEY=unused
> CHAT_MODEL=openchat
> EMBEDDING_PROVIDER=openchat
> LOCAL_IP=http://host.docker.internal
> VECTOR_SIZE=4096
> ```

i did according your config, but i got the below reply from bot copilot 
"HTTPConnectionPool(host='localhost', port=11434): Max retries exceeded with url: /api/generate/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f59863faaf0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
i dont know why this connection refused?
